RELEASE_NOTES.rst

```diff
@@ -21,6 +21,555 @@
 
 .. towncrier release notes start
 
+Airflow 2.3.0 (2022-04-29)
+--------------------------
+
```

Comment:
```suggestion

For production docker image related changes, see `Docker Image Changelog <https://github.com/apache/airflow/blob/main/docs/docker-stack/changelog.rst#airflow-23>`_

```
@potiuk WDYT

Reply:
```suggestion
For production docker image related changes, see the `Docker Image Changelog <https://airflow.apache.org/docs/docker-stack/changelog.html>`_.
```

Reply:
Yeah. The released in docs is better :)

Reply:
BTW. I am working on and update to the image build scripts to make some parameters a bit clearer - for refreshing the image and to re-add building the image for dockerhub with multi-platform image. 

It has no impact on airflow package/code just the scripts to build image and the Dockerfile itself: https://github.com/apache/airflow/pull/23267

I hope we will be able to merge it before cutting the branch :)

Reply:
I think it shoudl build now - pushed the version that has a good chance to work.